### ISSUE TITLE
Publish charts on Artifacthub

### ISSUE THREAD

**Issue by teemow** [giantswarm/roadmap#2427]
Publish our helm chart on artifacthub properly.

Would also be good to become a verified publisher.

Original thread: https://gigantic.slack.com/archives/CLPMFRVU6/p1681457734303279

**Comment by teemow** [comment]
@JosephSalisbury I've created an issue from the note. Do you have plans already?

**Comment by teemow** [comment]
Yeah. Depends on the amount of work. Would be good to stay top of mind.

**Comment by weatherhog** [comment]
Heads-up from team/honeybadger: we've just refined #3940, which covers the same goal (publish our charts on Artifact Hub, incl. Verified Publisher). It now contains the full research (public vs. private charts, OCI-first strategy from gsoci aligned with the HelmRelease migration, scriptable registration via the AH API) and a phased plan.

Proposal: consolidate on #3940 and close this one — but that's your call. Happy to sync if planeteers have additional requirements we should fold in.

**Comment by teemow** [comment]
Closing as completed. Done by Honey Badger in roadmap#3940 (weatherhog 2026-07-24: charts published on Artifact Hub, tested and announced). Evidence: https://github.com/giantswarm/roadmap/issues/3940 .
